chrono/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+from django.contib.auth.models import User
+
+
+class BaseModel(models.Model):
```

Comment:
Move this model somewhere other than the `chrono` directory. 

As a better practice, somewhere like `apps/contrib(or utils or extra or lib...)/models.py`